pandas/core/reshape/pivot.py

```diff
@@ -58,7 +58,9 @@ def pivot_table(
             pieces.append(table)
             keys.append(getattr(func, "__name__", func))
 
-        return concat(pieces, keys=keys, axis=1)
+        result = concat(pieces, keys=keys, axis=1)
```

Comment:
can concat be overloaded to avoid this?

even without Literal for axis could try List[DataFrame] -> DataFrame and List[Series} -> Union[Series, DataFrame]

Reply:
Yeah, that might work. I'll look into it.